src/pages/reader/pdf.vue

```diff
@@ -83,6 +83,19 @@ export default {
           this.fileData = null;
         }
       }
+      if (event.data.event === 'pdfDownload') {
+        logTrackerEvent(
+          this,
+          'ReaderPdf',
+          'PDFDownloaded',
+          {
+            classId: this.classId,
+            filename: event.data.filename,
+          },
+          1
+        );
```

Comment:
4th param does not expect object
and maybe we should name it as present tense since download is not completed at this point

```suggestion
        logTrackerEvent(
          this,
          'ReaderPdf',
          'ReaderDownloadPdf',
          classId: this.classId,
          1
        );
```